assets/src/edit-story/components/checklistCard/stories/index.js

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { ChecklistCard } from '..';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  THEME_CONSTANTS,
+  Text,
+  Link,
+  List,
+} from '../../../../design-system';
+import { Thumbnail, THUMBNAIL_TYPES } from '../../thumbnail';
+import { THUMBNAIL_BG } from '../../thumbnail/stories/demoThumbnails';
+import { CARD_TYPE } from '../constants';
+import { CardListWrapper } from '../styles';
+
+export default {
+  title: 'Stories Editor/Components/ChecklistCard',
+  component: ChecklistCard,
+};
+
+const Container = styled.div`
+  background-color: ${({ theme }) => theme.colors.bg.primary};
+  display: flex;
+  flex-wrap: wrap;
+  padding: 16px;
+  width: 900px;
+
+  & > div {
+    margin: 0 8px 16px;
+  }
+`;
+
+export const _default = () => {
+  return (
+    <Container>
+      <div>
+        <Text>{'Single Issue'}</Text>
+        <ChecklistCard
+          title="Add video captions"
+          titleProps={{
+            onClick: () => action('title clicked')(),
+          }}
+          footer={
+            <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+              {
+                'Keep the audience engaged even when they can’t listen to the audio. '
+              }
+              <Link
+                href="/demo"
+                size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+              >
+                {'Learn more'}
+              </Link>
+            </Text>
+          }
+          cta={
+            <Button size={BUTTON_SIZES.SMALL} type={BUTTON_TYPES.SECONDARY}>
+              {'Replace File'}
+            </Button>
```

Comment:
For elements like the `Text` & `Button` here I wonder if it's worth making like:
`FooterText` & `CardCta` which are just thin wrappers around the existing `Button` & `Text` components, but with `size` & `type` defaults already supplied. Just thinking if they all are gonna have the same presets could save having to import these constants in every time. That being, as you have it now totally works too.

Reply:
i thought about that and didn't do it because of edge cases where the cta's actually a checkbox or the footer has a list and needs a bonus wrapper because of the list type. :( we could have some defaults that we can pull in as wrappers as you suggest and then when we need to we can veer off. I'll add that in so when this is merged you have them.

Reply:
Right on. Yea it's no biggie, was just wondering if it'd be more convenient or not

Reply:
added :) if they are useful, awesome, if not we can do something else!